### PR TITLE
Update cloud-init feature page with IPv6 fields.

### PR DESCRIPTION
**Pull request by yevgenyz**
RHBZ#1330217 - [RFE] Enable configuring IPv6 in VM cloud-init

Changes proposed in this pull request:
- IPv6 fields were added to the example XML and that was updated to the current structure
- The screenshot is updated with the current one
- Update the case of the picture links

This pull request needs review by: @dankenigsberg @shaharha

### CODE REVIEW

**Comment by ILpinto** [comment]
I am ok with the feature page and adding IPV6 to cloud init.
As QA we will add new cases for IPV6 (and implement them in automation in 4.2)

File: source/develop/release-management/features/cloud/cloud-init-integration.html.md [hunk 1]
```diff
@@ -60,9 +62,9 @@ TODO: still pending is whether and/or how to communicate to oVirt engine that cl
 There is a long list of configuration options supported by cloud-init. The ones we are currently targeting for inclusion in oVirt include:
 
 *   set root password
-*   set timezone\*
+*   set timezone
```
Comment (mykaul) [inline source/develop/release-management/features/cloud/cloud-init-integration.html.md]: It's actually 'timezone' - see https://github.com/number5/cloud-init/blob/master/doc/examples/cloud-config.txt#L382

**Comment by mykaul** [comment]
@JohnMarksRH - no need to ACK changes, just comment on those with issues. 
I think most are small issues and we can merge and probably follow-up with cosmetics later.
@yevgenyz - anyone from your team approved?

File: source/develop/release-management/features/cloud/cloud-init-integration.html.md [hunk 2]
```diff
@@ -60,9 +62,9 @@ TODO: still pending is whether and/or how to communicate to oVirt engine that cl
...
 *   add ssh authorized keys for root user
-*   set static networking (ip/mask/gw/etc)\*
+*   set static IP (address/mask/gateway) for both IPv4 and IPv6\* stacks
```
Comment (mmirecki) [inline source/develop/release-management/features/cloud/cloud-init-integration.html.md]: Do we need the asterix?

Comment (yevgenyz) [inline source/develop/release-management/features/cloud/cloud-init-integration.html.md]: Done

**Pull request merged**